baselines/dom.generated.d.ts

```diff
@@ -7536,11 +7536,12 @@ declare var HashChangeEvent: {
 interface History {
     readonly length: number;
     readonly state: any;
-    back(distance?: any): void;
-    forward(distance?: any): void;
-    go(delta?: any): void;
-    pushState(statedata: any, title?: string, url?: string): void;
-    replaceState(statedata: any, title?: string, url?: string): void;
+    scrollRestoration: ScrollRestoration;
+    back(): void;
+    forward(): void;
+    go(delta?: number): void;
+    pushState(data: any, title: string, url?: string | null): void;
```

Comment:
Its signature in spec is `void pushState(any data, DOMString title, optional USVString? url = null);`. So it needs to accept a `null` value and should be optional.

https://html.spec.whatwg.org/multipage/browsers.html#the-history-interface